### PR TITLE
Fix support for typed subscriptions for multiple event names for `.on()`

### PR DESCRIPTION
**Pull request by arthur-er**
Closes #82

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -292,7 +295,9 @@ declare class Emittery<
 	emitter.emit('🐶', '🍖'); // Nothing happens
 	```
 	*/
-	once<Name extends keyof AllEventData>(eventName: Name): Promise<AllEventData[Name]>;
+	once<Name extends keyof AllEventData>(
+		eventName: Name
+	): Promise<AllEventData[Name]>;
```
Comment (sindresorhus) [inline index.d.ts:300]: Don't do unrelated changes.

Comment (arthur-er) [reply index.d.ts:300]: Sorry, it was auto run by Prettier, I will remove the unrelated changes

**Comment by sindresorhus** [comment]
Can you add it to all the methods that accept it? See: https://github.com/sindresorhus/emittery/pull/82/files

**Comment by arthur-er** [comment]
> Can you add it to all the methods that accept it? See: https://github.com/sindresorhus/emittery/pull/82/files

I'm unsure about how to add the tests. But I added the type into the other methods

**Comment by sindresorhus** [comment]
They need to be tested for this to be merged.

**Comment by arthur-er** [comment]
> They need to be tested for this to be merged.

Tests added!

**Comment by sindresorhus** [comment]
Tests are failing. Please always run `nom test` locally before pushing changes.

**Comment by arthur-er** [comment]
I'm getting issues with pEvent compatibility, the emitter in pEvent don't expect the .on() method to receive an array
